webpack.config.js

```diff
@@ -6,6 +6,7 @@ var index = require('./lib/index-static.jsx');
 
 module.exports = {
   entry: './lib/main.jsx',
+  devtool: 'source-map',
```

Comment:
Hmm, I just noticed that the exception tracebacks shown in mocha browser tests are also un-sourcemapped when using `source-map`, **even in Chrome**, but they work fine with `eval`.